python/lsst/sims/maf/batches/__init__.py

```diff
@@ -1,5 +1,6 @@
 from .colMapDict import *
 from .common import *
+from .srdBatch import *
 from .slewBatch import *
 from .timeBatch import *
 from .metadataBatch import *
```

Comment:
Line 9 of this _init__ has from .schedulerValidationBundle import * commented out. Is this still a new batch that will developed, or should it be removed before merging back to master.

Reply:
Yeah, I should just remove it. I don't think we're going to make a batch that exactly recreates the schedulerValidation as it is now.